src/views/TechnicalAssistance/AdminHome/Consult.tsx

```diff
@@ -196,6 +235,7 @@ function Consult() {
                   if (val) field.onChange(val);
```

Comment:
Removing the `if (val)` conditional will allow it to be `undefined` and appropriately trigger the validation